docker-compose-services/postgres/README.md

```diff
@@ -52,6 +52,58 @@ hooks:
 There are also another non-plain-text formats that `pg_dump` can generate, and you might need to work with them. If that's the case, there is also
 a `ddev pg_restore` command that will restore `.ddev/pgsql-db/postgresql.db.dump` into `db`.
 
+## Multiple databases
+
+* By default, DDEV adds a single database called `db`. But it's possible to create additional databases when you first initialize this service.
+
+* The offical way to add multiple databases is outlined on the [docker image](https://hub.docker.com/_/postgres) under the "Initialization scripts"
+
+To implement this in DDEV:
+
+* Add an additional volume to `./.ddev/docker-composer.postgres.yaml`
+
+  ```yaml
+      volumes:
+      ...
```

Comment:
This might be confusing, better to put the example without it

Reply:
Writing comments is not my strong point. English is hard.